### DISCUSSION TITLE
How do I change default location of appendonlydir and perl5 in linux?

### DISCUSSION THREAD

**Discussion by Crehet** [Difegue/LANraragi#748] [Q&A]
ideally i would like to move both folders to `$HOME/Github/Servers/`

cut and paste just wont do lol

**Comment by Difegue** [comment]
👋 Appendonlydir needs to be modified in the Redis config file.  

To use a different install of perl5, I'd recommend using perlbrew: https://perlbrew.pl/

**Reply by Crehet** [reply]
>Appendonlydir needs to be modified in the Redis config file.

you mean in `/etc/redis/redis.conf` 
````
appendonlydirname "appendonlydir"
````
and changing `appendonlydir` to `/home/user/Github/Servers/appendonlydir`?
then doing:
````
$ sudo redis-server /etc/redis/redis.conf
`````
>To use a different install of perl5, I'd recommend using perlbrew: https://perlbrew.pl/

overly complex for a weeb like me.

anyways, the one that bothers me the most is appendonlydir this folder gets created anywhere you are.

**Reply by Difegue** [reply]
That sounds about accurate. It's a bit weird that it'd create the AOF dir in your current directory, default redis configs usually don't do that. Ah well ¯\\\_(ツ)_/¯

**Reply by Crehet** [reply]
i was trying to reinstall lanraragi again. in my home folder there is no `~/.perl5` or `~/perl5` directory, and i've got [perl](https://archlinux.org/packages/core/x86_64/perl/) and [perlbrew](https://archlinux.org/packages/extra/any/perlbrew/) both installed.

unfortunately setting envs like: `export PERLBREW_ROOT=/opt/perl5` and `export PERL5LIB=~/Github/Servers/perl5/lib/perl5` then running `npm run lanraragi-installer install-full` didn't yield the result i was hoping for.
